src/NServiceBus.Persistence.AzureTable/SagaPersisters/AzureStorageSagaDefaults.cs

```diff
@@ -2,9 +2,9 @@
 {
     static class AzureStorageSagaDefaults
     {
-        public const bool CreateSchema = true;
         public const bool AssumeSecondaryIndicesExist = true;
         public const bool AssumeSecondaryKeyUsesANonEmptyRowKeySetToThePartitionKey = false;
         public const bool CompatibilityModeEnabled = true;
+        public const string ConventionalTablePrefix = null;
```

Comment:
Null or empty string?
Null could feel as not assigned. Empty string is a value and would have the same effect.

Reply:
Empty string is fine for me as well. I think you have a good point about the assignment